### DISCUSSION TITLE
Change request - Expose C3 CN in evidence

### DISCUSSION THREAD

**Discussion by AzetsInsightAS** [OxalisCommunity/oxalis#669] [General]
As a sender, it would be great if the Common Name of C3 (receiver, in this case) could be exposed in the evidence. 
This information is already a part of the metadata (BinarySecurityToken) in the receipt, but it is a bit of a hassle to extract the necessary data. 

It would be great if the CN of C3 recipient could be made available directly in the evidence for easy extraction for further use in the PEPPOL Statistics Reporting, as it would eliminate the need for a separate SML Lookup to fetch the ID.

**Comment by aaron-kumar** [comment]
@AzetsInsightAS  I hope you have gone through documentation in Wiki: https://github.com/OxalisCommunity/oxalis/wiki/Peppol-Reporting.  You can retrieve all fields the way it is suggested in documentation. 
Evidence is well defined structure and it can't be modified.

**Comment by AzetsInsightAS** [comment]
@aaron-kumar Thank you for the input. I couldn't find the [example for the logger](https://github.com/OxalisCommunity/oxalis/wiki/Peppol-Reporting#sample-outbound-reporting-fields-code-snippet) last time around when I finalized the reporting in December/January; and I apologize for the oversight. I'll look into this. 

Thanks again.
